### DISCUSSION TITLE
Can't actived copilot in VS 2022,When I typed in activation code, and nothing happened in Vs 2022

### DISCUSSION THREAD

**Discussion by 15091071931** [community/community#14103] [Copilot Conversations]
Can't actived copilot in VS 2022,When I typed in activation code, and nothing happened in Vs 2022
![image](https://user-images.githubusercontent.com/59719005/161389372-27159caf-aa93-4f9e-9e1e-a9684c0921c8.png)

**Comment by kzu** [comment]
After you enter the activation code, VS will show a telemetry acceptance message box. If that isn't the case, can you start VS with `devenv /log` so you can get extra logging in the GitHub Copilot output window pane for troubleshooting?

**Reply by vimcaw** [reply]
```
GitHub Copilot 1.12.0.0 (v1.12.0.0@0660f7705)
10:31:30.472 GitHub Copilot Agent v1.12.0
10:31:30.703 [Verbose] LSP [Loaded (62112816)]
10:31:31.407 [Verbose] LSP [Activated (62112816)]
10:31:31.423 [Verbose] LSP [Attached (62112816)]
10:31:31.426 [Verbose] LSP [C:\Users\PC\AppData\Local\Temp\VSLogs\VSB33F9F54.LSPClient.Copilot.JKOY.svclog]
10:31:31.426 [Verbose] LSP [C:\Users\PC\AppData\Local\Temp\VisualStudio\LSP\CopilotLanguageClient-62112816.log]
10:31:31.528 [Verbose] LSP [Initialized (62112816)]
10:31:31.529 [Verbose] CheckStatusAsync duration='869ms' (Background)
10:31:37.255 [Verbose] SignInInitiateAsync duration='454ms' (Background)
10:31:37.285 A device activation code 5766-FF9E has been copied to the clipboard.
```
It doesn't help, no extra info after authorization.

**Comment by vimcaw** [comment]
For anyone live in China mainland, GFW cause instability to access GitHub, you can open [TUN Mode](https://docs.cfw.lbyczf.com/contents/tun.html) on `Clash For Windows`.

**Comment by BillDong2021** [comment]
Same!!! what can I do? 
一样，搞了一天还没解决，该咋办啊

**Reply by vimcaw** [reply]
看我上面回复的，开了增强模式（ClashX Pro）或者 TUN 模式（Clash For Windows）吗？
